evalml/tests/component_tests/test_column_selector_transformers.py

```diff
@@ -105,39 +106,22 @@ def test_drop_column_transformer_input_invalid_col_name(class_to_test):
 
 
 @pytest.mark.parametrize("class_to_test,answers",
-                         [(DropColumns, [np.array([[0, 2, 3], [4, 6, 7], [8, 10, 11]]),
-                                         np.array([[], [], []]),
-                                         np.arange(12).reshape(3, 4)]),
-                          (SelectColumns, [np.array([[1], [5], [9]]),
-                                           np.arange(12).reshape(3, 4),
-                                           np.array([[], [], []])])
+                         [(DropColumns, [pd.DataFrame([[0, 2, 3], [4, 6, 7], [8, 10, 11]], columns=[0, 2, 3], dtype="Int64"),
+                                         pd.DataFrame([[], [], []], dtype="Int64"),
+                                         pd.DataFrame(np.arange(12).reshape(3, 4), dtype="Int64")]),
+                          (SelectColumns, [pd.DataFrame([[1], [5], [9]], columns=[1], dtype="Int64"),
+                                           pd.DataFrame(np.arange(12).reshape(3, 4), dtype="Int64"),
+                                           pd.DataFrame([[], [], []], dtype="Int64")])
                           ])
-def test_column_transformer_numpy(class_to_test, answers):
+def test_column_transformer_int_col_names(class_to_test, answers):
     X = np.arange(12).reshape(3, 4)
     answer1, answer2, answer3 = answers
 
     transformer = class_to_test(columns=[1])
-    np.testing.assert_allclose(transformer.transform(X).values, answer1)
+    assert_frame_equal(answer1, transformer.transform(X).to_dataframe())
 
     transformer = class_to_test(columns=[0, 1, 2, 3])
-    np.testing.assert_allclose(transformer.transform(X).values, answer2)
+    assert_frame_equal(answer2, transformer.transform(X).to_dataframe())
 
     transformer = class_to_test(columns=[])
-    np.testing.assert_allclose(transformer.transform(X).values, answer3)
-
-
-@pytest.mark.parametrize("class_to_test,answers",
```

Comment:
I think this test is redundant with the former, since we're using a numpy array, and numpy array will automatically have int col names! So removing and combining.